src/FieldWarning/Assets/Units/UnitBehaviour.cs

```diff
@@ -19,13 +19,27 @@
 public abstract class UnitBehaviour : SelectableBehavior, Matchable<Vector3>
 {
     public const string UNIT_TAG = "Unit";
+    private const float ORIENTATION_RATE = 5.0f;
+    private const float TRANSLATION_RATE = 10.0f;
 
     public UnitData Data = UnitData.GenericUnit();
     public PlatoonBehaviour platoon { get; private set; }
     public bool IsAlive { get; private set; }
     public Pathfinder pathfinder { get; private set; }
     public AudioSource source { get; private set; }
 
+    // These are set by the subclass in DoMovement()
+    protected Vector3 position;
+    protected Vector3 rotation;
```

Comment:
_targetRotation? (especially for clarity in the subclass, where with the current naming it would seem like you are directly applying the rotation instead of setting a target)

Reply:
There are already a bunch of things named 'target', though. The entire movement system probably needs to have a lot of the variables renamed.